hoomd/md/EvaluatorPairALJ.h

```diff
@@ -283,7 +283,14 @@ template<unsigned int ndim> class EvaluatorPairALJ
             // Simulating an ellipsoid just return
             if (N_vertices == 0 && N_faces == 0)
                 {
-                faces = ManagedArray<unsigned int>(0, managed);
+                // for ellipsoids, there needs to be a 0 vertex
+                // because that is the 0 of the support function
```

Comment:
```suggestion
                // The GJK implementation assumes that all shapes have at least one
                // vertex. For ellipsoids, we must use the origin as that vertex since it
                // guarantees that the polyhedral component of the support function will
                // always be 0 and not contribute to the total support function.
```